src/tests/Microsoft.PowerFx.Interpreter.Tests/RecalcEngineTests.cs

```diff
@@ -767,6 +764,21 @@ public void TestNumberBinding()
             Assert.True(recalcEngine.DefineFunctions("Foo(): String => 10;").Errors.Any());
         }
 
+        [Fact]
+        public void TestWithTimeZoneInfo()
+        {
+            var pfxConfig = new PowerFxConfig(new CultureInfo("ja-JP"));
+            var recalcEngine = new RecalcEngine(pfxConfig);
+            var symbols = new SymbolValues();
+
+            symbols.AddService(TimeZoneInfo.FindSystemTimeZoneById("Romance Standard Time")); // (UTC+01:00) Brussels, Copenhagen, Madrid, Paris            
+            var fv = recalcEngine.Eval(@"DateTimeValue(""jeudi 21 juillet 2022 19:34:03"", ""fr-FR"")", symbols: symbols);

```

Comment:
>Eval [](http://example.com/codeflow?start=34&length=4)

Does this expression actually use the TimeZoneInfo?

Reply:
Nope, I need to update that test to use it and demonstrate its effect